tests/detect-vlan-id/README.md

```diff
@@ -0,0 +1,3 @@
+Test for checking the working of vlan.id keyword by creating rules and matching a crafted packet against them. The packet is an ICMP packet with 3 different VLAN ids [200,300,400].
```

Comment:
As with the docs, these look better if we keep lines to a certain character count limit.
Let's try around 79 or 80?